src/plugins/data_source_management/public/components/data_source_view/_data_source_view.scss

```diff
@@ -0,0 +1,3 @@
+.panelWidth {
+  width: 300px;
+}
```

Comment:
Let's follow guideline for style file and class name https://github.com/opensearch-project/OpenSearch-Dashboards/actions/runs/8667700544/job/23771246546?pr=6416